doc/analysis-plan.md

```diff
@@ -0,0 +1,54 @@
+# Possible analyses
+### I. How to answer research questions 
+For variable community composition, we plan to plot them with PCA and spatial autocorrelation, identify dominant genus (simpson's index), abundance/importance (if have over 10 genus).
```

Comment:
Can you say that we will test for spatial autocorrelation of community composition? Plotting the autocorrelation isn't what we are interested in i think.  Also simpson's index is a measure of diversity and idk if we can say that it is also a measure of community composition.